include/tachimawari/control/manager/control_manager.hpp

```diff
@@ -50,6 +51,7 @@ class ControlManager
   virtual void disconnect() {}
 
   virtual bool ping(uint8_t id) {}
+  virtual bool ping_core(uint8_t id) {}
```

Comment:
You can also remove this like you remove the `ping_core` method in its deriving class.

Reply:
updated